src/lib/y2users/password.rb

```diff
@@ -18,20 +18,29 @@
 # find current contact information at www.suse.com.
 
 require "yast2/execute"
+require "yast2/secret_attributes"
```

Comment:
Please, adapt the spec file in order to depend on the new *yast2* that provides *secret_attributes*.